### PR TITLE
Profile view screen

### PR DESCRIPTION
**Pull request by mpace965**
its the profile

screenshot_20180223-000658

also contains some patches like when the private key leaked in the errors

### CODE REVIEW

File: client/app/screens/ProfileScreen.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+class ProfileScreen extends React.Component {
+  state = {
+    loading: true
+  };
+
+  componentDidMount() {
+    const { navigation } = this.props;
+
+    this.props.fetchProfileAsync(navigation.state.params.userId)
```
Comment (TomFanella4) [inline client/app/screens/ProfileScreen.js]: This can be destructured above

File: client/app/screens/ProfileScreen.js [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+            </CardItem>
+            {positions}
+          </Card>
+        </Content>
+      );
+    }
+
+    return (
+      <Container style={styles.container}>
+        <Content>
```
Comment (TomFanella4) [inline client/app/screens/ProfileScreen.js]: Duplicate Content components (Line 50)

File: client/app/actions/index.js [hunk 3]
```diff
@@ -35,6 +42,35 @@ export const fetchMatchesAsync = userId => {
...
+  return async dispatch => {
+    let profile;
+
+    if (useMocks) {
+      const uri = 'https://meetover.herokuapp.com/test/liprofile';
+      const init = { method: 'POST' };
+
+      const response = await fetch(uri, init);
+      profile = await response.json();
+      profile = JSON.parse(profile);
```
Comment (TomFanella4) [inline client/app/actions/index.js:60]: Is this necessary? The JSON should already be parsed in line 59

Comment (mpace965) [reply client/app/actions/index.js:60]: Yes. Talked offline. This is a server side problem, can likely be addressed with the content type

**Pull request merged**